Networking/BLIP/BLIPConnection.cc

```diff
@@ -154,6 +155,16 @@ namespace litecore::blip {
 
         WebSocket* webSocket() const { return _webSocket; }
 
+        Retained<WebSocket> webSocketSafe() const {
```

Comment:
Why don't we just name this method `webSocket()`, i.e. replace the existing method with this one. There's no need to have two when there's only one caller of this method.

Reply:
I was meaning to make as little change as possible. This is a very special case. In general, the webSocket is like a private member of connection. In fact except for uses by tests, Replicator::onConnect is the only place where webSocket is used. And it's used only to get role(). If we store the role in Connection at the construction time, we don't even need to expose it.
However, as you pointed, since we are exposing it, why not to expose only safe version.
I will make the change.